### ISSUE TITLE
Adding pointer-events: none when playing youtube video

### ISSUE THREAD

**Issue by SabatonShip** [dyc3/opentogethertube#761]
### Is your feature request related to a problem? Please describe.

I am very infuriated by movable objects when moving the mouse across the screen while watching a video, it infuriates that when you click on a video while watching it will be put on a phantom pause for 1 second, it infuriates that when you click on the video chat will not open when you click on the letter T. I propose the following solution:

### Describe the solution you'd like

Can you please add something like pointer-events: none to the youtube-video div? Or add an invisible div on top of youtube-video div?

### Additional context

https://www.youtube.com/watch?v=JdO00-fR5G0

**Comment by dyc3** [comment]
Yeah, this is an unfortunate reality of using Youtube's iframe API. When you click on the iframe video, your browser switches focus to that iframe. This makes all keyboard events impossible to intercept by the parent frame, which means all of OTT's global keybinds don't work. The workaround is to click somewhere outside of the video to change the focus back to OTT.

Currently, we do have some workarounds regarding mouse move behavior and dealing with iframes.

That being said, I really don't want to ignore pointer events on iframes, because it's sometimes necessary to interact with the video directly in order to play it. Placing an element in front of the iframe or making the iframe itself ignore pointer events creates the potential to cause even more frustration. I have been thinking about ditching youtube's iframe player altogether, or at least making it optional, but for now it's functional so it's staying.

**Comment by SabatonShip** [comment]
> Yeah, this is an unfortunate reality of using Youtube's iframe API. When you click on the iframe video, your browser switches focus to that iframe. This makes all keyboard events impossible to intercept by the parent frame, which means all of OTT's global keybinds don't work. The workaround is to click somewhere outside of the video to change the focus back to OTT.
> 
> Currently, we do have some workarounds regarding mouse move behavior and dealing with iframes.
> 
> That being said, I really don't want to ignore pointer events on iframes, because it's sometimes necessary to interact with the video directly in order to play it. Placing an element in front of the iframe or making the iframe itself ignore pointer events creates the potential to cause even more frustration. I have been thinking about ditching youtube's iframe player altogether, or at least making it optional, but for now it's functional so it's staying.

Ok thanks, this is sad
